src/node_os.cc

```diff
@@ -123,15 +123,20 @@ static void GetCPUInfo(const FunctionCallbackInfo<Value>& args) {
 
     Local<Object> times_info = Object::New(env->isolate());
     times_info->Set(env->user_string(),
-                    Number::New(env->isolate(), ci->cpu_times.user));
+                    Number::New(env->isolate(),
+                      static_cast<double>(ci->cpu_times.user)));
     times_info->Set(env->nice_string(),
-                    Number::New(env->isolate(), ci->cpu_times.nice));
+                    Number::New(env->isolate(),
+                      static_cast<double>(ci->cpu_times.nice)));
     times_info->Set(env->sys_string(),
-                    Number::New(env->isolate(), ci->cpu_times.sys));
+                    Number::New(env->isolate(),
+                      static_cast<double>(ci->cpu_times.sys)));
     times_info->Set(env->idle_string(),
-                    Number::New(env->isolate(), ci->cpu_times.idle));
+                    Number::New(env->isolate(),
+                      static_cast<double>(ci->cpu_times.idle)));
     times_info->Set(env->irq_string(),
-                    Number::New(env->isolate(), ci->cpu_times.irq));
+                    Number::New(env->isolate(),
+                      static_cast<double>(ci->cpu_times.irq)));
```

Comment:
Style: arguments should line up but lines should not exceed 80 columns.  Worst case, you can write it as:

```c++
times_info->Set(
    env->irq_string(),
    Number::New(...));
```